install-agora-git.sh

```diff
@@ -182,4 +182,9 @@ nohup.out
 GNUstep
 EOF
 
+
+[ -x /usr/local/etc ] || (sudo mkdir -p /usr/local/etc/profile.d)
+[ -x /usr/local/etc/profile ] || (sudo ln -sf /etc/profile /usr/local/etc/profile)
+[ -x /usr/local/etc/profile.d/GNUstep.sh ] || (sudo ln -sf `gnustep-config --variable=GNUSTEP_MAKEFILES`/GNUstep.sh /usr/local/etc/profile.d/GNUstep.sh)
```

Comment:
`-x` checks that the target is executable. I think you want `-d` for the directory, and `-f` for the files/links.

```suggestion
[ -d /usr/local/etc ] || (sudo mkdir -p /usr/local/etc/profile.d)
[ -f /usr/local/etc/profile ] || (sudo ln -sf /etc/profile /usr/local/etc/profile)
[ -f /usr/local/etc/profile.d/GNUstep.sh ] || (sudo ln -sf `gnustep-config --variable=GNUSTEP_MAKEFILES`/GNUstep.sh /usr/local/etc/profile.d/GNUstep.sh)
```

https://en.wikibooks.org/wiki/Bourne_Shell_Scripting/Control_flow

Reply:
DOH, too late

Reply:
#23

Reply:
Accepted with thanks